site/_posts/2017-12-13-java-vector-improvements.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: post
+title: "Improved JAVA Vector APIs"
+excerpt: "This post describes the recent improvements in JAVA Vector code"
+date: 2017-12-13 12:50:00
+author: Siddharth Teotia
+categories: [application]
+---
+
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+This post gives insight into the major improvements in the JAVA implementation
+of vectors.
+
+## Design Goals
+
+1. Improved Maintainability and Extensibility.
```

Comment:
lower case `Maintainability` and `Extensibility`?

Reply:
Done.